app/views/trips/_form.html.erb

```diff
@@ -41,6 +40,11 @@
     <%= form.number_field :pets %>
   </div>
 
+  <div>
+    <%= form.label :vehicle_id, style: "display: block" %>
+    <%= form.select(:vehicle_id, options_for_select(Vehicle.user_vehicles(current_user).map { |elm| [elm.nickname, elm.id] })) %>
```

Comment:
maybe it's easier doing something like `options_for_select(current_user.vehicles.select(:nickanem, :id))`